ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/management_security.schema.yml

```
@@ -32,6 +32,11 @@ keys:
               type: str
               description: |
                 List of allowed TLS versions as string
+                Examples:
+                  - "1.0"
+                  - "1.0 1.1"
```

Are we not supporting the below one?

```suggestion
                Examples:
                  - "1.0"
                  - "1.0 1.1"
                  - 1.2
```

---

The auto conversions are to help the user. The correct input type is string, so our examples are only covering that.